.github/workflows/run-helm.yml

```diff
@@ -40,6 +40,23 @@ jobs:
 
       - uses: azure/setup-helm@v1
 
+      # We should generate two sets of akri charts.
+      # One for our releases (akri) and one for our merged PRs (akri-dev).  'akri' should only be created for
+      # our releases and should default to not use dev containers.  'akri-dev'
+      # should build for any merged PR and should default to use our dev containers.
+      - name: Split into akri and akri-dev charts (this step is for release == akri)
+        if: (github.event_name == 'release')
+        run: |
+          sed -i s/"name: akri"/"name: akri"/g ./deployment/helm/Chart.yaml
```

Comment:
`s/"name: akri/"name: akri"/g` doesn't do anything ... some of these sed commands aren't needed, but i left them in for (what i thought was) clarity.  happy to remove if others find it confusing.